app/assets/stylesheets/friendships.css.scss

```diff
@@ -0,0 +1,15 @@
+.friendship {
+  border-bottom: 1px dotted lightgrey;
```

Comment:
Color `lightgrey` should be written in hexadecimal form as `#d3d3d3`<br>Color literals like `lightgrey` should only be used in variable declarations; they should be referred to via variable everywhere else.